examples/gnrc_networking/README.md

```diff
@@ -96,10 +95,9 @@ Now, on the RIOT side, send a UDP packet using:
 
 You should see `testmessage` appear in netcat. Instead of using netcat,
 you can of course write your own software, but you may have to bind the
-socket to a specific interface (tap0 in this case). For an example that
-shows how to do so, see [here][tap-socket]
-
-[tap-socket]: (https://gist.github.com/backenklee/dad5e80b764b3b3d0d3e).
+socket to a specific interface (tap0 in this case). To do so, have a
+look at [setting the `SO_BINDTODEVICE` option using `setsocketopt()`
+](https://stackoverflow.com/questions/14478167/bind-socket-to-network-interface#14478657).
```

Comment:
I think we still need the reflink ;-).

Reply:
I agree, but I couldn't get it to work? If you look at https://github.com/RIOT-OS/RIOT/blob/master/examples/gnrc_networking/README.md, it's broken there too. I'll give it another try tomorrow. :)

Reply:
Maybe the reference definition *has* to be at the bottom of the document...

Reply:
Ah, nope, we just got the syntax wrong 😬 . Fixed it now (sorry for the cacophony of trial & error fixups). Still put the link in the bottom because the file looks cleaner that way.

Reply:
Ah, nope, we just got the syntax wrong 😬 . Fixed it now (sorry for the cacophony of trial & error fixups). Still put the link in the bottom because the file looks cleaner that way.